debian/kolibri-server.init

```diff
@@ -9,7 +9,10 @@
 # Description:       Multicore server setup for Kolibri, an offline education platform
 ### END INIT INFO
 
-# Do NOT "set -e"
+# Crash if a sub-process crashes. This is important to discover
+# errors in this script. It might be changed in the future.
+set -e
```

Comment:
I'd prefer to revert this change, it's gonna give us more headaches than benefits.

Reply:
Sure but I'll have to go back to testing it, to make sure that the system service reports an error if UWSGI fails to start.